pkg/machines/libvirt.es6

```diff
@@ -226,10 +226,16 @@ function spawnVirshReadOnly({connectionName, method, name, failHandler}) {
 function parseDumpxml(dispatch, connectionName, domXml) {
     const xmlDoc = $.parseXML(domXml);
 
+    if (!xmlDoc) {
+        console.error(`Can't parse dumpxml, input: "${domXml}"`);
+        return ;
+    }
```

Comment:
This really should not happen :-) 
It means, the `virsh dumpxml` produced unparsable output or there was error in transfer - I noticed transfer of an empty string.
With this "handling", subsequent data retrieval is executed so there's still a chance to get reasonable data and during next polling interval the `dumpxml` is reexecuted.

Other option is to fail as before this change, so an exception is thrown and the cockpit's error messages in the right top corner are populated.
I would not introduce other error messages printed on the page.